algebird-core/src/main/scala/com/twitter/algebird/Interval.scala

```diff
@@ -283,11 +290,9 @@ case class Intersection[L[t] <: Lower[t], U[t] <: Upper[t], T](lower: L[T], uppe
    * it this way, it does not mean it is empty or universe, etc... (there
    * are other cases).
    */
-  def toLeftClosedRightOpen(implicit s: Successible[T]): Option[Intersection[InclusiveLower, ExclusiveUpper, T]] = {
-    implicit val ord = lower.ordering
+  def toLeftClosedRightOpen(implicit s: Successible[T]): Option[Intersection[InclusiveLower, ExclusiveUpper, T]] =
```

Comment:
Let's try to see if we can just jump Successible and Predecessible up to Ordering. I can't think of a good reason to have Partial there.